test/evergreen.yml

```diff
@@ -1033,14 +1033,15 @@ tasks:
   - name: million-collection-test
     depends_on: []
     run_on:
-      - rhel62-large
+      - rhel80-build
```

Comment:
Not related to the change directly - above 3 lines seem to be no-ops and could be cleaned up, as there's no dependent task and the build variant level has the same distro setting.